packages/checkout/widgets-lib/src/widgets/swap/SwapWidget.tsx

```diff
@@ -179,7 +179,8 @@ export function SwapWidget({
       const exchange = new Exchange({
         chainId: network.chainId,
         baseConfig: new ImmutableConfiguration({ environment }),
-        overrides,
+        secondaryFees: dexConfig.secondaryFees,
```

Comment:
This will initialise the Exchange class with any secondary fees that are included in the fetched configuration